Core/Models/ProviderModel.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace Core.Models
+{
+    public class ProviderModel
+    {
+        [Key]
+        public Guid _id { get; set; }
+
+        public string login { get; set; }
+
+        public string password { get; set; }
+
+        public string ImapHost { get; set; }
```

Comment:
O ile małych liter nie rozumiem to nie trzymania się ich konsekwentnie nie rozumiem jeszcze bardziej